tests/phpunit/tests/block-bindings/render.php

```diff
@@ -234,4 +234,41 @@ public function test_source_value_with_unsafe_html_is_sanitized() {
 			'The block content should be updated with the value returned by the source.'
 		);
 	}
+
+	/**
+	 * Tests if the block content is sanitized when unsafe HTML is passed.
+	 *
+	 * @ticket 61333
+	 *
+	 * @covers ::register_block_bindings_source
+	 */
+	public function test_default_binding_for_pattern_overrides() {
+		$get_value_callback = function ( $source_args, $block_instance, $attribute_name ) {
+			$value = $source_args['key'];
+			return "The attribute name is '$attribute_name'";
+		};
+
+		register_block_bindings_source(
+			'core/pattern-overrides',
```

Comment:
Not sure if pattern overrides will already be registered in the test environment 🤔

Reply:
It doesn't seem to be, but the test is still failing. I'm struggling to understand why 🤔

Reply:
> It doesn't seem to be, but the test is still failing. I'm struggling to understand why 🤔

Actually it is. So I've updated the test to use the actual pattern overrides source and now it seems to be passing.